e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -433,15 +433,152 @@ test.describe('Example imagery thumbnails resize in display layouts', () => {
 
     });
 
-});
+    // test.fixme('Can use Mouse Wheel to zoom in and out of previous image');
+    // test.fixme('Can use alt+drag to move around image once zoomed in');
+    // test.fixme('Can zoom into the latest image and the real-time/fixed-time imagery will pause');
+    // test.fixme('Clicking on the left arrow should pause the imagery and go to previous image');
+    // test.fixme('If the imagery view is in pause mode, it should not be updated when new images come in');
+    // test.fixme('If the imagery view is not in pause mode, it should be updated when new images come in');
+    const backgroundImageSelector = '.c-imagery__main-image__background-image';
+    test('Example Imagery in Flexible layout', async ({ page }) => {
+        // Go to baseURL
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        // Click the Create button
+        await page.click('button:has-text("Create")');
+
+        // Click text=Flexible Layout
+        await page.click('text=Flexible Layout');
+
+        // Assert Flexable layout
+        await expect(page.locator('.js-form-title')).toHaveText('Create a New Flexible Layout');
+
+        // Click OK button
+        page.click('text=OK');
+
+        //Wait for Save Banner to appear
+        page.waitForSelector('.c-message-banner__message');
+
+        // Wait until Save Banner is gone
+        await page.waitForSelector('.c-message-banner__message', { state: 'detached'});
+        await expect(page.locator('.l-browse-bar__object-name')).toContainText('Unnamed Flexible Layout');
+
+        // Click My Items
+        page.locator('.c-disclosure-triangle').click();
+
+        // !!!!!!!!!!!!!!!!! Missing images, can't move forward till we have images here!!!!!!!!!!!!!!!!
```

Comment:
@unlikelyzero I am not seeing any images here, <y Images is empty.
<img width="1312" alt="Screen Shot 2022-05-21 at 4 22 23 PM" src="https://user-images.githubusercontent.com/24461216/169667975-3a9fceb2-85a2-4caf-b4d2-5a914651a659.png">
Any advice to have at least one image in here?

Reply:
@afahey87 yes, you will need to create exampleimagery object and drag into one of the Flex Layout Panels